src/index.ts

```diff
@@ -1,5 +1,5 @@
-import * as jd from './astro/jd';
-import * as math from './math';
+import * as jd from './astro/jd.js';
```

Comment:
There is no js anymore:)

Reply:
yes but adding the `js` extension make it work in typescript and javascript
https://github.com/nodejs/node/issues/46006

Reply:
Bad:( I don't like this solution. Need to think. @tbo47, did you face it in your projects and always resolve it like this?